--- /home/asus/asuswrt-merlin/release/src/router/rc/firewall.c	2017-01-05 10:56:44.934046331 -0500
+++ /home/asus/asuswrt-merlin/release/ecrs/router/rc/mods/firewall.c	2017-01-04 10:18:24.631509220 -0500
@@ -1301,6 +1301,12 @@
 	}
 #endif
 
+        // ECRS - If hht kiosk rule is enabled then add nat rule
+        if (nvram_get_int("ecrs_hht_kiosk_enable"))
+        {
+                fprintf(fp, "-A PREROUTING -d 10.218.1.1 -j DNAT --to-destination %s\n", nvram_safe_get("ecrs_hht_kiosk_ip"));
+        }
+
 #ifdef RTCONFIG_YANDEXDNS
 	if (nvram_get_int("yadns_enable_x"))
 		write_yandexdns_filter(fp, lan_if, lan_ip);
@@ -1601,6 +1607,12 @@
 
 	}
 
+        // ECRS - If hht kiosk rule is enabled then add nat rule
+        if (nvram_get_int("ecrs_hht_kiosk_enable"))
+        {
+                fprintf(fp, "-A PREROUTING -d 10.218.1.1 -j DNAT --to-destination %s\n", nvram_safe_get("ecrs_hht_kiosk_ip"));
+        }
+
 #ifdef RTCONFIG_YANDEXDNS
 	if (nvram_get_int("yadns_enable_x"))
 		write_yandexdns_filter(fp, lan_if, lan_ip);
