--- /home/asus/asuswrt-merlin/release/ecrs/router/rc/original/firewall.c	2016-11-03 09:11:19.848707996 -0400
+++ /home/asus/asuswrt-merlin/release/ecrs/router/rc/mods/firewall.c	2016-11-08 12:05:29.408095041 -0500
@@ -1244,6 +1244,12 @@
 	}
 #endif
 
+        // ECRS - If hht kiosk rule is enabled then add nat rule
+        if (nvram_get_int("ecrs_hht_kiosk_enable"))
+        {
+                fprintf(fp, "-A PREROUTING -d 10.218.1.1 -j DNAT --to-destination %s\n", nvram_safe_get("ecrs_hht_kiosk_ip"));
+        }
+
 #ifdef RTCONFIG_YANDEXDNS
 	if (nvram_get_int("yadns_enable_x"))
 		write_yandexdns_filter(fp, lan_if, lan_ip);
@@ -1524,6 +1530,12 @@
 
 	}
 
+        // ECRS - If hht kiosk rule is enabled then add nat rule
+        if (nvram_get_int("ecrs_hht_kiosk_enable"))
+        {
+                fprintf(fp, "-A PREROUTING -d 10.218.1.1 -j DNAT --to-destination %s\n", nvram_safe_get("ecrs_hht_kiosk_ip"));
+        }
+
 #ifdef RTCONFIG_YANDEXDNS
 	if (nvram_get_int("yadns_enable_x"))
 		write_yandexdns_filter(fp, lan_if, lan_ip);
