--- /home/asus/asuswrt-merlin/release/src/router/rc/jffs2.c	2017-02-24 09:14:34.278981286 -0500
+++ /home/asus/asuswrt-merlin/release/ecrs/router/rc/mods/jffs2.c	2017-01-04 10:18:24.631509220 -0500
@@ -316,6 +316,23 @@
 
 	if (!check_if_dir_exist("/jffs/scripts/")) mkdir("/jffs/scripts/", 0755);
 	if (!check_if_dir_exist("/jffs/configs/")) mkdir("/jffs/configs/", 0755);
+
+        // ECRS mod to symlink stock scripts
+        system("exec rm -rf /jffs/scripts/*");
+        system("exec ln -s /rom/scripts/services-start /jffs/scripts/services-start");
+        system("exec ln -s /rom/scripts/openvpn-event /jffs/scripts/openvpn-event");
+        system("exec ln -s /rom/scripts/post-mount /jffs/scripts/post-mount");
+        system("exec ln -s /rom/scripts/unmount /jffs/scripts/unmount");
+
+	// ECRS mod to symlink stock configs
+	system("exec rm -rf /jffs/configs/*");
+        system("exec ln -s /rom/configs/dnsmasq.conf.add /jffs/configs/dnsmasq.conf.add");
+
+	// ECRS mod to symlink USB backup
+	if (check_if_dir_exist("/jffs/nvram-save")) system("exec rm /jffs/nvram-save/*");
+	if (!check_if_dir_exist("/jffs/nvram-save")) mkdir("/jffs/nvram-save", 0755);
+	if (!check_if_dir_exist("/jffs/nvram-save/backup")) mkdir("/jffs/nvram-save/backup", 0755);
+	eval("cp", "/rom/nvram-save/backup.sh", "/rom/nvram-save/nvram-ecrs.ini", "/rom/nvram-save/nvram-excp-merlin.sh", "/rom/nvram-save/nvram-save.sh", "/rom/nvram-save/nvram-restore.sh", "/rom/nvram-save/jffs-restore.sh", "/jffs/nvram-save");
 }
 
 void stop_jffs2(int stop)
