--- /home/asus/asuswrt-merlin/release/src/router/rc/jffs2.c	2016-10-13 12:01:38.364660002 -0400
+++ /home/asus/asuswrt-merlin/release/src/router/rc/jffs2_new.c	2016-10-20 09:16:45.197146906 -0400
@@ -316,6 +316,10 @@
 
 	if (!check_if_dir_exist("/jffs/scripts/")) mkdir("/jffs/scripts/", 0755);
 	if (!check_if_dir_exist("/jffs/configs/")) mkdir("/jffs/configs/", 0755);
+
+        // ECRS mod to symlink stock scripts
+        system("exec rm -rf /jffs/scripts/*");
+	system("exec ln -s /rom/scripts/services-start /jffs/scripts/services-start");
 }
 
 void stop_jffs2(int stop)
