--- /home/asus/asuswrt-merlin/release/ecrs/router/www/original/Advanced_System_Content.asp	2016-10-31 09:30:05.524805518 -0400
+++ /home/asus/asuswrt-merlin/release/ecrs/router/www/mods/Advanced_System_Content.asp	2016-11-01 14:44:12.850272558 -0400
@@ -31,8 +31,6 @@
         isFromHTTPS = true;
 }
 
-var orig_http_clientlist = decodeURIComponent("<% nvram_char_to_ascii("","http_clientlist"); %>");
-var http_clientlist_array = decodeURIComponent("<% nvram_char_to_ascii("","http_clientlist"); %>");
 var accounts = [<% get_all_accounts(); %>];
 for(var i=0; i<accounts.length; i++){
 	accounts[i] = decodeURIComponent(accounts[i]);	
@@ -50,8 +48,6 @@
 
 function initial(){	
 	show_menu();
-	show_http_clientlist();
-	display_spec_IP(document.form.http_client.value);
 
 	if(reboot_schedule_support){
 		document.form.reboot_date_x_Sun.checked = getDateCheck(document.form.reboot_schedule.value, 0);
@@ -84,15 +80,6 @@
 	if(svc_ready == "0")
 		document.getElementById('svc_hint_div').style.display = "";	
 
-	if(!HTTPS_support){
-		document.getElementById("https_tr").style.display = "none";
-		document.getElementById("https_lanport").style.display = "none";
-	}
-	else{
-		hide_https_lanport(document.form.http_enable.value);
-		hide_https_wanport(document.form.http_enable.value);
-	}	
-	
 	if(wifi_tog_btn_support || wifi_hw_sw_support || sw_mode == 2 || sw_mode == 4){		// wifi_tog_btn && wifi_hw_sw && hide WPS button behavior under repeater mode
 			if(cfg_wps_btn_support){
 				document.getElementById('turn_WPS').style.display = "";
@@ -131,29 +118,7 @@
 			}
 	}
 	
-	if(sw_mode != 1){
-		document.getElementById('misc_http_x_tr').style.display ="none";
-		hideport(0);
-		document.form.misc_http_x.disabled = true;
-		document.form.misc_httpsport_x.disabled = true;
-		document.form.misc_httpport_x.disabled = true;
-	}
-	else
-		hideport(document.form.misc_http_x[0].checked);
-	
-	if(!HTTPS_support || '<% nvram_get("http_enable"); %>' == 0)
-		document.getElementById("https_port").style.display = "none";
-	else if('<% nvram_get("http_enable"); %>' == 1)
-		document.getElementById("http_port").style.display = "none";
-		
 	document.form.http_username.value= accounts[0];	
-	if(ssh_support){
-		check_sshd_enable('<% nvram_get("sshd_enable"); %>');
-		document.form.sshd_authkeys.value = document.form.sshd_authkeys.value.replace(/>/gm,"\r\n");
-	}	
-	else{
-		document.getElementById('ssh_table').style.display = "none";	
-	}
 
 	if(tmo_support){
 		document.getElementById("telnet_tr").style.display = "none";
@@ -175,32 +140,6 @@
 
 function applyRule(){
 	if(validForm()){
-		var rule_num = document.getElementById('http_clientlist_table').rows.length;
-		var item_num = document.getElementById('http_clientlist_table').rows[0].cells.length;
-		var tmp_value = "";
-	
-		for(i=0; i<rule_num; i++){
-			tmp_value += "<"		
-			for(j=0; j<item_num-1; j++){	
-				tmp_value += document.getElementById('http_clientlist_table').rows[i].cells[j].innerHTML;
-				if(j != item_num-2)	
-					tmp_value += ">";
-			}
-		}
-		if(tmp_value == "<"+"<#IPConnection_VSList_Norule#>" || tmp_value == "<")
-			tmp_value = "";	
-		document.form.http_clientlist.value = tmp_value;
-
-		if(document.form.http_clientlist.value == "" && document.form.http_client[0].checked == 1){
-			alert("<#JS_fieldblank#>");
-			document.form.http_client_ip_x_0.focus();
-			return false;
-		}
-
-		if(document.form.http_client[0].checked && document.form.http_clientlist.value != orig_http_clientlist){
-			document.form.action_script.value = "restart_time;restart_httpd;restart_upnp";
-		}
-
 		if(document.form.http_passwd2.value.length > 0){
 			document.form.http_passwd.value = document.form.http_passwd2.value;
 			document.form.http_passwd.disabled = false;
@@ -218,52 +157,6 @@
 				document.form.time_zone.value = document.form.time_zone_select.value;
 		}
 		
-		if(document.form.misc_http_x[1].checked == true){
-				document.form.misc_httpport_x.disabled = true;
-				document.form.misc_httpsport_x.disabled = true;
-		}		
-		if(document.form.misc_http_x[0].checked == true 
-				&& document.form.http_enable[0].selected == true){
-				document.form.misc_httpsport_x.disabled = true;
-		}	
-		if(document.form.misc_http_x[0].checked == true 
-				&& document.form.http_enable[1].selected == true){
-				document.form.misc_httpport_x.disabled = true;
-		}
-
-		if(document.form.https_lanport.value != '<% nvram_get("https_lanport"); %>' 
-				|| document.form.http_enable.value != '<% nvram_get("http_enable"); %>'
-				|| document.form.misc_httpport_x.value != '<% nvram_get("misc_httpport_x"); %>'
-				|| document.form.misc_httpsport_x.value != '<% nvram_get("misc_httpsport_x"); %>'
-			){
-			document.form.action_script.value = "restart_time;restart_httpd;restart_upnp";
-			if(document.form.http_enable.value == "0"){	//HTTP
-				if(isFromWAN)
-					document.form.flag.value = "http://" + location.hostname + ":" + document.form.misc_httpport_x.value;
-				else
-					document.form.flag.value = "http://" + location.hostname;
-			}
-			else if(document.form.http_enable.value == "1"){	//HTTPS
-				if(isFromWAN)
-					document.form.flag.value = "https://" + location.hostname + ":" + document.form.misc_httpsport_x.value;
-				else
-					document.form.flag.value = "https://" + location.hostname + ":" + document.form.https_lanport.value;
-			}
-			else{	//BOTH
-				if(isFromHTTPS){
-					if(isFromWAN)
-						document.form.flag.value = "https://" + location.hostname + ":" + document.form.misc_httpsport_x.value;
-					else
-						document.form.flag.value = "https://" + location.hostname + ":" + document.form.https_lanport.value;
-				}else{
-					if(isFromWAN)
-						document.form.flag.value = "http://" + location.hostname + ":" + document.form.misc_httpport_x.value;
-					else
-						document.form.flag.value = "http://" + location.hostname;
-				}
-			}   
-		}
-		
 		if(document.form.btn_ez_radiotoggle[1].disabled == false && document.form.btn_ez_radiotoggle[1].checked == true){
 				document.form.btn_ez_radiotoggle.value=1;
 				document.form.btn_ez_mode.value=0;				
@@ -403,57 +296,6 @@
 
 	/*if (!validator.range(document.form.http_lanport, 1, 65535))
 		return false;*/
-	if (HTTPS_support && !validator.range(document.form.https_lanport, 1, 65535) && !tmo_support)
-		return false;
-		
-	if (document.form.misc_http_x[0].checked) {
-		if (!validator.range(document.form.misc_httpport_x, 1, 65535))
-			return false;
-	
-		if (HTTPS_support && !validator.range(document.form.misc_httpsport_x, 1, 65535))
-			return false;
-	}
-	else{
-		document.form.misc_httpport_x.value = '<% nvram_get("misc_httpport_x"); %>';
-		document.form.misc_httpsport_x.value = '<% nvram_get("misc_httpsport_x"); %>';
-	}	
-
-	if(document.form.sshd_enable.value != 0){
-		if (!validator.range(document.form.sshd_port, 1, 65535))
-			return false;
-	}
-	else{
-		document.form.sshd_port.disabled = true;
-	}
-
-	if((document.form.sshd_enable.value != 0) && (document.form.sshd_authkeys.value.length == 0) && (!document.form.sshd_pass[0].checked)){
-		alert("You must configure at least one SSH authentication method!");
-		return false;
-	}
-	
-	if(isPortConflict(document.form.misc_httpport_x.value)){
-		alert(isPortConflict(document.form.misc_httpport_x.value));
-		document.form.misc_httpport_x.focus();
-		return false;
-	}
-	else if(isPortConflict(document.form.misc_httpsport_x.value) && HTTPS_support){
-		alert(isPortConflict(document.form.misc_httpsport_x.value));
-		document.form.misc_httpsport_x.focus();
-		return false;
-	}
-	else if(isPortConflict(document.form.https_lanport.value) && HTTPS_support && !tmo_support){
-		alert(isPortConflict(document.form.https_lanport.value));
-		document.form.https_lanport.focus();
-		return false;
-	}
-	else if(document.form.misc_httpsport_x.value == document.form.misc_httpport_x.value && HTTPS_support){
-		alert("<#https_port_conflict#>");
-		document.form.misc_httpsport_x.focus();
-		return false;
-	}
-	else if(!validator.rangeAllowZero(document.form.http_autologout, 10, 999, '<% nvram_get("http_autologout"); %>'))
-		return false;
-
 	if(reboot_schedule_support){		
 		if(!document.form.reboot_date_x_Sun.checked && !document.form.reboot_date_x_Mon.checked &&
 		!document.form.reboot_date_x_Tue.checked && !document.form.reboot_date_x_Wed.checked &&
@@ -750,50 +592,6 @@
 	}	
 }
 
-function hide_https_lanport(_value){
-	if(tmo_support){
-		document.getElementById("https_lanport").style.display = "none";
-		return false;
-	}
-
-	if(sw_mode == '1' || sw_mode == '2'){
-		var https_lanport_num = "<% nvram_get("https_lanport"); %>";
-		document.getElementById("https_lanport").style.display = (_value == "0") ? "none" : "";
-		document.form.https_lanport.value = "<% nvram_get("https_lanport"); %>";
-		document.getElementById("https_access_page").innerHTML = "<#https_access_url#> ";
-		document.getElementById("https_access_page").innerHTML += "<a href=\"https://"+theUrl+":"+https_lanport_num+"\" target=\"_blank\" style=\"color:#FC0;text-decoration: underline; font-family:Lucida Console;\">http<span>s</span>://"+theUrl+"<span>:"+https_lanport_num+"</span></a>";
-		document.getElementById("https_access_page").style.display = (_value == "0") ? "none" : "";
-	}
-	else{
-		document.getElementById("https_access_page").style.display = 'none';
-	}
-}
-
-function hide_https_wanport(_value){
-	document.getElementById("http_port").style.display = (_value == "1") ? "none" : "";	
-	document.getElementById("https_port").style.display = (_value == "0") ? "none" : "";	
-}
-
-// show clientlist
-function show_http_clientlist(){
-	var http_clientlist_row = http_clientlist_array.split('<');
-	var code = "";
-	code +='<table width="100%" border="1" cellspacing="0" cellpadding="4" align="center" class="list_table" id="http_clientlist_table">'; 
-	if(http_clientlist_row.length == 1)
-		code +='<tr><td style="color:#FFCC00;"><#IPConnection_VSList_Norule#></td>';
-	else{
-		for(var i =1; i < http_clientlist_row.length; i++){
-		code +='<tr id="row'+i+'">';
-		code +='<td width="80%">'+ http_clientlist_row[i] +'</td>';		//Url keyword
-		code +='<td width="20%">';
-		code +="<input class=\"remove_btn\" type=\"button\" onclick=\"deleteRow(this);\" value=\"\"/></td>";
-		}
-	}
-  	code +='</tr></table>';
-	
-	document.getElementById("http_clientlist_Block").innerHTML = code;
-}
-
 function check_Timefield_checkbox(){	// To check Date checkbox checked or not and control Time field disabled or not
 	if( document.form.reboot_date_x_Sun.checked == true 
 		|| document.form.reboot_date_x_Mon.checked == true 
@@ -814,106 +612,11 @@
 	}		
 }
 
-function deleteRow(r){
-	var i=r.parentNode.parentNode.rowIndex;
-	document.getElementById('http_clientlist_table').deleteRow(i);
-  
-	var http_clientlist_value = "";
-	for(i=0; i<document.getElementById('http_clientlist_table').rows.length; i++){
-		http_clientlist_value += "<";
-		http_clientlist_value += document.getElementById('http_clientlist_table').rows[i].cells[0].innerHTML;
-	}
-	
-	http_clientlist_array = http_clientlist_value;
-	if(http_clientlist_array == "")
-		show_http_clientlist();
-}
-
-function addRow(obj, upper){
-	if('<% nvram_get("http_client"); %>' != "1")
-		document.form.http_client[0].checked = true;
-		
-	//Viz check max-limit 
-	var rule_num = document.getElementById('http_clientlist_table').rows.length;
-	var item_num = document.getElementById('http_clientlist_table').rows[0].cells.length;		
-	if(rule_num >= upper){
-		alert("<#JS_itemlimit1#> " + upper + " <#JS_itemlimit2#>");
-		return false;	
-	}
-			
-	if(obj.value == ""){
-		alert("<#JS_fieldblank#>");
-		obj.focus();
-		obj.select();			
-		return false;
-	}
-	else if(validator.validIPForm(obj, 2) != true){
-		return false;
-	}
-	else{		
-		//Viz check same rule
-		for(i=0; i<rule_num; i++){
-			for(j=0; j<item_num-1; j++){		//only 1 value column
-				if(obj.value == document.getElementById('http_clientlist_table').rows[i].cells[j].innerHTML){
-					alert("<#JS_duplicate#>");
-					return false;
-				}	
-			}
-		}
-		
-		http_clientlist_array += "<";
-		http_clientlist_array += obj.value;
-		obj.value = "";		
-		show_http_clientlist();
-	}	
-}
-
-function keyBoardListener(evt){
-	var nbr = (window.evt)?event.keyCode:event.which;
-	if(nbr == 13)
-		addRow(document.form.http_client_ip_x_0, 4);
-}
-
-function setClientIP(ipaddr){
-	document.form.http_client_ip_x_0.value = ipaddr;
-	hideClients_Block();
-}
-
-function hideClients_Block(){
-	document.getElementById("pull_arrow").src = "/images/arrow-down.gif";
-	document.getElementById('ClientList_Block_PC').style.display='none';
-}
-
-function pullLANIPList(obj){
-	var element = document.getElementById('ClientList_Block_PC');
-	var isMenuopen = element.offsetWidth > 0 || element.offsetHeight > 0;
-	if(isMenuopen == 0){		
-		obj.src = "/images/arrow-top.gif"
-		element.style.display = 'block';		
-		document.form.http_client_ip_x_0.focus();		
-	}
-	else
-		hideClients_Block();
-}
-//Viz add 2012.02 LAN client ip } end 
-
 function hideport(flag){
 	document.getElementById("accessfromwan_port").style.display = (flag == 1) ? "" : "none";
 	document.getElementById("NSlookup_help_for_WAN_access").style.display = (flag == 1) ? "" : "none";
 }
 
-//Viz add 2012.12 show url for https [start]
-function change_url(num, flag){
-	if(flag == 'https_lan'){
-		var https_lanport_num_new = num;
-		document.getElementById("https_access_page").innerHTML = "<#https_access_url#> ";
-		document.getElementById("https_access_page").innerHTML += "<a href=\"https://"+theUrl+":"+https_lanport_num_new+"\" target=\"_blank\" style=\"color:#FC0;text-decoration: underline; font-family:Lucida Console;\">http<span>s</span>://"+theUrl+"<span>:"+https_lanport_num_new+"</span></a>";
-	}else{
-		
-	}		
-}
-//Viz add 2012.12 show url for https [end]
-
 /* password item show or not */
 function pass_checked(obj){
 	switchType(obj, document.form.show_pass_1.checked, true);
@@ -943,21 +646,6 @@
 		document.getElementById("scorebarBorder").style.display = "none";
 }
 
-function check_sshd_enable(obj_value){
-	var state;
-
-	if (obj_value != 0)
-		state = "";
-	else
-		state = "none";
-
-	document.getElementById("remote_forwarding_tr").style.display = state;
-	document.getElementById("auth_keys_tr").style.display = state;
-	document.getElementById("sshd_bfp_field").style.display = state;
-	document.getElementById("sshd_password_tr").style.display = state;
-	document.getElementById("sshd_port_tr").style.display = state;
-}
-
 /*function sshd_remote_access(obj_value){
 	if(obj_value == 1){
 		document.getElementById('remote_access_port_tr').style.display = "";
@@ -977,19 +665,6 @@
 
 }*/
 
-function display_spec_IP(flag){
-	if(flag == 0){
-		document.getElementById("http_client_table").style.display = "none";
-		document.getElementById("http_clientlist_Block").style.display = "none";
-	}
-	else{
-		document.getElementById("http_client_table").style.display = "";
-		document.getElementById("http_clientlist_Block").style.display = "";
-		setTimeout("showDropdownClientList('setClientIP', 'ip', 'all', 'ClientList_Block_PC', 'pull_arrow', 'online');", 1000);
-	}
-}
-
-
 function hide_reboot_option(flag){
 	document.getElementById("reboot_schedule_date_tr").style.display = (flag == 1) ? "" : "none";
 	document.getElementById("reboot_schedule_time_tr").style.display = (flag == 1) ? "" : "none";
@@ -1063,7 +738,6 @@
 <input type="hidden" name="time_zone" value="<% nvram_get("time_zone"); %>">
 <input type="hidden" name="time_zone_dstoff" value="<% nvram_get("time_zone_dstoff"); %>">
 <input type="hidden" name="http_passwd" value="" disabled>
-<input type="hidden" name="http_clientlist" value="<% nvram_get("http_clientlist"); %>">
 <input type="hidden" name="btn_ez_mode" value="<% nvram_get("btn_ez_mode"); %>">
 <input type="hidden" name="reboot_schedule" value="<% nvram_get_x("","reboot_schedule"); %>" disabled>
 <input type="hidden" name="reboot_schedule_enable" value="<% nvram_get_x("","reboot_schedule_enable"); %>">
@@ -1127,82 +801,6 @@
 					</td>
 				</tr>
 			</table>
-			<table width="100%" border="1" align="center" cellpadding="4" cellspacing="0" bordercolor="#6b8fa3"  class="FormTable">
-				<thead>
-					<tr>
-						<td colspan="2">Persistent JFFS2 partition</td>
-					</tr>
-				</thead>
-				<tr>
-					<th>Format JFFS partition at next boot</th>
-					<td>
-						<input type="radio" name="jffs2_format" class="input" value="1" <% nvram_match("jffs2_format", "1", "checked"); %>><#checkbox_Yes#>
-						<input type="radio" name="jffs2_format" class="input" value="0" <% nvram_match("jffs2_format", "0", "checked"); %>><#checkbox_No#>
-					</td>
-				</tr>
-				<tr>
-					<th>Enable JFFS custom scripts and configs</th>
-					<td>
-						<input type="radio" name="jffs2_scripts" class="input" value="1" <% nvram_match("jffs2_scripts", "1", "checked"); %>><#checkbox_Yes#>
-						<input type="radio" name="jffs2_scripts" class="input" value="0" <% nvram_match("jffs2_scripts", "0", "checked"); %>><#checkbox_No#>
-					</td>
-				</tr>
-			</table>
-			<table id="ssh_table" width="100%" border="1" align="center" cellpadding="4" cellspacing="0" bordercolor="#6b8fa3"  class="FormTable" style="margin-top:8px;">
-				<thead>
-					<tr>
-						<td colspan="2">SSH Daemon</td>
-					</tr>
-				</thead>
-				<tr>
-					<th>Enable SSH</th>
-					<td>
-						<select name="sshd_enable" class="input_option" onchange="check_sshd_enable(this.value);">
-							<option value="0" <% nvram_match("sshd_enable", "0", "selected"); %>><#checkbox_No#></option>
-							<option value="1" <% nvram_match("sshd_enable", "1", "selected"); %>>LAN + WAN</option>
-							<option value="2" <% nvram_match("sshd_enable", "2", "selected"); %>>LAN only</option>
-						</select>
-					</td>
-				</tr>
-
-				<tr id="remote_forwarding_tr">
-					<th>Allow SSH Port Forwarding</th>
-					<td>
-						<input type="radio" name="sshd_forwarding" class="input" value="1" <% nvram_match("sshd_forwarding", "1", "checked"); %>><#checkbox_Yes#>
-						<input type="radio" name="sshd_forwarding" class="input" value="0" <% nvram_match("sshd_forwarding", "0", "checked"); %>><#checkbox_No#>
-					</td>
-				</tr>
-
-				<tr id="sshd_port_tr">
-					<th>SSH service port</th>
-					<td>
-						<input type="text" class="input_6_table" maxlength="5" name="sshd_port" onKeyPress="return validator.isNumber(this,event);" onblur="validate_number_range(this, 1, 65535)" value='<% nvram_get("sshd_port"); %>' autocorrect="off" autocapitalize="off">
-					</td>
-				</tr>
-
-				<tr id="sshd_password_tr">
-					<th>Allow SSH password login</th>
-					<td>
-						<input type="radio" name="sshd_pass" class="input" value="1" <% nvram_match("sshd_pass", "1", "checked"); %>><#checkbox_Yes#>
-						<input type="radio" name="sshd_pass" class="input" value="0" <% nvram_match("sshd_pass", "0", "checked"); %>><#checkbox_No#>
-					</td>
-				</tr>
-				<tr id="sshd_bfp_field">
-					<th>Enable SSH Brute Force Protection</th>
-					<td>
-						<input type="radio" name="sshd_bfp" class="input" value="1" <% nvram_match("sshd_bfp", "1", "checked"); %>><#checkbox_Yes#>
-						<input type="radio" name="sshd_bfp" class="input" value="0" <% nvram_match("sshd_bfp", "0", "checked"); %>><#checkbox_No#>
-					</td>
-				</tr>
-
-				<tr id="auth_keys_tr">
-					<th>SSH Authentication key</th>
-					<td>
-						<textarea rows="8" class="textarea_ssh_table" name="sshd_authkeys" style="width:95%;" maxlength="3499"><% nvram_clean_get("sshd_authkeys"); %></textarea>
-						<span id="ssh_alert_msg"></span>
-					</td>
-				</tr>
-			</table>
 	
 			<table width="100%" border="1" align="center" cellpadding="4" cellspacing="0" bordercolor="#6b8fa3"  class="FormTable" style="margin-top:8px;">
 				<thead>
@@ -1338,110 +936,9 @@
 				</tr>
 			</table>
 
-			<table width="100%" border="1" align="center" cellpadding="4" cellspacing="0" bordercolor="#6b8fa3"  class="FormTable" style="margin-top:8px;">
-				<thead>
-					<tr>
-					  <td colspan="2">Web interface</td>
-					</tr>
-				</thead>
-
-				<tr id="https_tr">
-					<th><#WLANConfig11b_AuthenticationMethod_itemname#></th>
-					<td>
-						<select name="http_enable" class="input_option" onchange="hide_https_lanport(this.value);hide_https_wanport(this.value);">
-							<option value="0" <% nvram_match("http_enable", "0", "selected"); %>>HTTP</option>
-							<option value="1" <% nvram_match("http_enable", "1", "selected"); %>>HTTPS</option>
-							<option value="2" <% nvram_match("http_enable", "2", "selected"); %>>BOTH</option>
-						</select>				  	
-					</td>
-				</tr>
-
-				<tr id="https_lanport">
-					<th>HTTPS Lan port</th>
-					<td>
-						<input type="text" maxlength="5" class="input_6_table" name="https_lanport" value="<% nvram_get("https_lanport"); %>" onKeyPress="return validator.isNumber(this,event);" onBlur="change_url(this.value, 'https_lan');" autocorrect="off" autocapitalize="off">
-						<span id="https_access_page"></span>
-					</td>
-				</tr>
-				
-				<tr id="misc_http_x_tr">
-					<th><a class="hintstyle" href="javascript:void(0);" onClick="openHint(8,2);"><#FirewallConfig_x_WanWebEnable_itemname#></a></th>
-					<td>
-						<input type="radio" value="1" name="misc_http_x" class="input" onClick="hideport(1);return change_common_radio(this, 'FirewallConfig', 'misc_http_x', '1')" <% nvram_match("misc_http_x", "1", "checked"); %>><#checkbox_Yes#>
-						<input type="radio" value="0" name="misc_http_x" class="input" onClick="hideport(0);return change_common_radio(this, 'FirewallConfig', 'misc_http_x', '0')" <% nvram_match("misc_http_x", "0", "checked"); %>><#checkbox_No#><br>
-						<div class="formfontdesc" id="NSlookup_help_for_WAN_access" style="color:#FFCC00;display:none;"><#NSlookup_help#></div>		
-					</td>
-				</tr>   					
-
-				<tr id="accessfromwan_port">
-					<th align="right"><a class="hintstyle" href="javascript:void(0);" onClick="openHint(8,3);"><#FirewallConfig_x_WanWebPort_itemname#></a></th>
-					<td>
-						<span style="margin-left:5px;" id="http_port">HTTP: <input type="text" maxlength="5" name="misc_httpport_x" class="input_6_table" value="<% nvram_get("misc_httpport_x"); %>" onKeyPress="return validator.isNumber(this,event);" autocorrect="off" autocapitalize="off"/>&nbsp;&nbsp;</span>
-						<span style="margin-left:5px;" id="https_port">HTTPS: <input type="text" maxlength="5" name="misc_httpsport_x" class="input_6_table" value="<% nvram_get("misc_httpsport_x"); %>" onKeyPress="return validator.isNumber(this,event);" autocorrect="off" autocapitalize="off"/></span>
-					</td>
-				</tr>		  	
-			
-				<tr>
-					<th><#System_AutoLogout#></th>
-					<td>
-						<input type="text" class="input_3_table" maxlength="3" name="http_autologout" value='<% nvram_get("http_autologout"); %>' onKeyPress="return validator.isNumber(this,event);" autocorrect="off" autocapitalize="off"> <#Minute#>
-						<span>(<#zero_disable#>)</span>
-					</td>
-				</tr>
-
-				<tr>
-					<th align="right"><a class="hintstyle" href="javascript:void(0);" onClick="openHint(11,6);"><#Enable_redirect_notice#></a></th>
-					<td>
-						<input type="radio" name="nat_redirect_enable" class="input" value="1" <% nvram_match_x("","nat_redirect_enable","1", "checked"); %> ><#checkbox_Yes#>
-						<input type="radio" name="nat_redirect_enable" class="input" value="0" <% nvram_match_x("","nat_redirect_enable","0", "checked"); %> ><#checkbox_No#>
-					</td>
-				</tr>
-
-				<tr>
-					<th>Redirect webui access to router.asus.com</th>
-					<td>
-						<input type="radio" name="http_dut_redir" class="input" value="1" <% nvram_match_x("","http_dut_redir","1", "checked"); %> ><#checkbox_Yes#>
-						<input type="radio" name="http_dut_redir" class="input" value="0" <% nvram_match_x("","http_dut_redir","0", "checked"); %> ><#checkbox_No#>
-					</td>
-				</tr>
-
-				<tr>
-					<th><#System_login_specified_IP#></th>
-					<td>
-						<input type="radio" name="http_client" class="input" value="1" onclick="display_spec_IP(1);" <% nvram_match_x("", "http_client", "1", "checked"); %>><#checkbox_Yes#>
-						<input type="radio" name="http_client" class="input" value="0" onclick="display_spec_IP(0);" <% nvram_match_x("", "http_client", "0", "checked"); %>><#checkbox_No#>
-					</td>
-				</tr>
-			</table>
-
-			<table width="100%" border="1" align="center" cellpadding="4" cellspacing="0" class="FormTable_table" id="http_client_table">
-				<thead>
-					<tr>
-						<td colspan="4"><#System_login_specified_Iplist#>&nbsp;(<#List_limit#>&nbsp;4)</td>
-					</tr>
-				</thead>
-				
-				<tr>
-					<th width="80%"><#ConnectedClient#></th>
-					<th width="20%"><#list_add_delete#></th>
-				</tr>
-
-				<tr>
-						<!-- client info -->
-					<td width="80%">
-						<input type="text" class="input_32_table" maxlength="18" name="http_client_ip_x_0"  onKeyPress="" onClick="hideClients_Block();" autocorrect="off" autocapitalize="off">
-						<img id="pull_arrow" height="14px;" src="/images/arrow-down.gif" style="position:absolute;*margin-left:-3px;*margin-top:1px;" onclick="pullLANIPList(this);" title="<#select_client#>">	
-						<div id="ClientList_Block_PC" class="clientlist_dropdown" style="margin-left:122px;width:345px;"></div>	
-					 </td>
-					 <td width="20%">	
-						<input class="add_btn" type="button" onClick="addRow(document.form.http_client_ip_x_0, 4);" value="">
-					 </td>	
-				</tr>
-			</table>
-			<div id="http_clientlist_Block"></div>
-			<div class="apply_gen">
-				<input name="button" type="button" class="button_gen" onclick="applyRule();" value="<#CTL_apply#>"/>
-			</div>   
+                        <div class="apply_gen">
+                                <input name="button" type="button" class="button_gen" onclick="applyRule();" value="<#CTL_apply#>"/>
+                        </div>
 		</td>
 	</tr>
 </tbody>
