--- /home/asus/asuswrt-merlin/release/src/router/www/EN.dict	2017-01-27 14:13:23.090291852 -0500
+++ /home/asus/asuswrt-merlin/release/ecrs/router/www/mods/EN.dict	2017-01-04 10:18:24.631509220 -0500
@@ -2531,3 +2531,4 @@
 Facebook_WiFi_unit_hint1=All 5 GHz guest networks are used, if you want to use 5 GHz band for Facebook Wi-Fi, please remove one 5 GHz guest network<br>
 Facebook_WiFi_unit_hint2=All 5 GHz-2 guest networks are used, if you want to use 5 GHz band for Facebook Wi-Fi, please remove one 5 GHz-2 guest network
 Facebook_WiFi_unit_hint_all=All guest networks are used now, please remove at least one guest network then try to enable Facebook Wi-Fi again.
+ECRS_Backup_USB_itemdesc=External USB medium with volume label 'ASUS' must be plugged into the router in order for USB backup to be enabled.
