--- /home/asus/asuswrt-merlin/release/src/router/www/EN.dict	2017-02-21 07:54:41.737560340 -0500
+++ /home/asus/asuswrt-merlin/release/ecrs/router/www/mods/EN.dict	2017-02-21 06:38:57.549439606 -0500
@@ -135,8 +135,8 @@
 JS_validstr2=This string cannot contain:
 JS_wepkey=Please enter the correct WEP key.
 JS_wpapass=Pre-shared key should be less than 64 characters!
-LAN_IP_changed_suggedtion1=The LAN IP of ZVMODELVZ is
-LAN_IP_changed_suggedtion2=.If you can't get back setting page later, please renew IP configuration of your computer.
+LAN_IP_changed_suggedtion1=The LAN IP of your ZVMODELVZ is 
+LAN_IP_changed_suggedtion2=. If you cannot access your ZVMODELVZ via this IP address you may need to renew the IP of your computer.
 LANConfig_ChangedLANIP=The IP address for your router has been changed. Please use the Device Discovery utility to search for your router's new IP address.
 LANHostConfig_display1_sectionname=LAN IP Setting
 LANHostConfig_IPRouters_itemdesc=The LAN IP address of ZVMODELVZ. The default value is 192.168.1.1. In IP-based networks, data packets are sent to the network devices' specific IP addresses.
@@ -2531,3 +2531,4 @@
 Facebook_WiFi_unit_hint1=All 5 GHz guest networks are used, if you want to use 5 GHz band for Facebook Wi-Fi, please remove one 5 GHz guest network<br>
 Facebook_WiFi_unit_hint2=All 5 GHz-2 guest networks are used, if you want to use 5 GHz band for Facebook Wi-Fi, please remove one 5 GHz-2 guest network
 Facebook_WiFi_unit_hint_all=All guest networks are used now, please remove at least one guest network then try to enable Facebook Wi-Fi again.
+ECRS_Backup_USB_itemdesc=External USB medium with volume label 'ASUS' must be plugged into the router in order for USB backup to be enabled.
