--- /home/asus/asuswrt-merlin/release/src/router/www/state.js	2017-02-02 14:25:06.827148764 -0500
+++ /home/asus/asuswrt-merlin/release/ecrs/router/www/mods/state.js	2017-01-04 10:18:24.635509454 -0500
@@ -314,7 +314,7 @@
 var dualWAN_support = isSupport("dualwan");
 var ruisp_support = isSupport("ruisp");
 var ssh_support = isSupport("ssh");
-var snmp_support = isSupport("snmp");
+var snmp_support = 0;
 var multissid_support = isSupport("mssid");
 var no5gmssid_support = rc_support.search("no5gmssid");
 var no5gmssid_support = isSupport("no5gmssid");
@@ -460,7 +460,7 @@
 var gn_array_5g_2 = <% wl_get_guestnetwork("2"); %>;
 
 //notification value
-var notice_pw_is_default = '<% check_pw(); %>';
+var notice_pw_is_default = 0;
 if(notice_pw_is_default == 1 && window.location.pathname.toUpperCase().search("QIS_") < 0)	//force to change http_passwd / http_username & except QIS settings
 		location.href = 'Main_Password.asp?nextPage=' + window.location.pathname.substring(1 ,window.location.pathname.length);
 var noti_auth_mode_2g = '<% nvram_get("wl0_auth_mode_x"); %>';
@@ -653,7 +653,7 @@
 
 	banner_code +='<div id="banner1" class="banner1" align="center"><img src="images/New_ui/asustitle.png" width="218" height="54" align="left">\n';
 	banner_code +='<div style="margin-top:13px;margin-left:-90px;*margin-top:0px;*margin-left:0px;" align="center"><span id="modelName_top" onclick="this.focus();" class="modelName_top"><#Web_Title2#></span></div>';
-	banner_code +='<div style="margin-left:25px;width:160px;margin-top:0px;float:left;" align="left"><span><a href="http://asuswrt.lostrealm.ca/" target="_blank"><img src="images/merlin-logo.png" style="border: 0;"></span></div>';
+	banner_code +='<div style="margin-left:25px;width:160px;margin-top:5px;float:left;" align="left"><span><a href="https://ecrs.com" target="_blank"><img src="images/ecrs_asuswrt.svg" style="border: 0;"></span></div>';
 
 	// logout, reboot
 	banner_code +='<a href="javascript:logout();"><div style="margin-top:13px;margin-left:25px; *width:136px;" class="titlebtn" align="center"><span><#t1Logout#></span></div></a>\n';
@@ -1761,7 +1761,40 @@
 			menu1_code += '<div class="m_qis_r" style="margin-top:-170px;cursor:pointer;" onclick="go_setting(\''+ QISWIZARD +'?flag=wireless\');"><table><tr><td><div id="index_img0"></div></td><td><div><#QIS#></div></td></tr></table></div>\n';
 		else
 			menu1_code += '<div class="m_qis_r" style="margin-top:-170px;cursor:pointer;" onclick="go_setting(\''+ QISWIZARD +'?flag=detect\');"><table><tr><td><div id="index_img0"></div></td><td><div><#QIS#></div></td></tr></table></div>\n';
-	}	
+	}
+
+	// ECRS - Backup status
+	var ecrs_backup_usb_enable = '<% nvram_get("ecrs_backup_usb_enable"); %>';
+	if (ecrs_backup_usb_enable == 1)
+	{
+		var ecrs_backup_usb_status = '<% nvram_get("ecrs_backup_usb_status"); %>';
+		var color = '', text1 = '', text2 = '';
+		switch (parseInt(ecrs_backup_usb_status))
+		{
+			case -1:
+				color = 'orange';
+				text1 = 'PENDING';
+				text2 = 'Awaiting first backup';
+				break;
+			case 0:
+				color = 'green';
+				text1 = 'SUCCESS';
+				text2 = '<% nvram_get("ecrs_backup_usb_text"); %>';
+				break;
+			case 1:
+				color = 'red';
+				text1 = 'FAILURE';
+				text2 = '<% nvram_get("ecrs_backup_usb_text"); %>';
+				break;
+			default:
+				color = 'yellow';
+				text1 = 'UNKNOWN';
+				text2 = '<% nvram_get("ecrs_backup_usb_text"); %>';
+		}
+
+		menu1_code += '<div style="margin:10px 0px 0px 10px;font-weight:bold;"><span style="color:white;">Backup Status:</span> <span style="color:' + color + '";>' + text1 + '</span></div>';
+		menu1_code += '<div style="margin: 0px 10px;">' + text2 + '</div>';
+	}
 
 	// Feature
 	menu1_code += '<div class="m0_r" style="margin-top:10px;" id="option0"><table width="192px" height="37px"><tr><td><#menu5_1_1#></td></tr></table></div>\n';
@@ -2376,13 +2409,12 @@
 
 	// FAQ searching bar{
 	footer_code += '<div style="margin-top:-75px;margin-left:205px;"><table width="765px" border="0" align="center" cellpadding="0" cellspacing="0"><tr>';
-	footer_code += '<td width="20" align="right"><div id="bottom_help_icon" style="margin-right:3px;"></div></td><td width="115" id="bottom_help_title" align="left"><#Help#> & <#Support#></td>';
+	footer_code += '<td width="20" align="right"><div id="bottom_help_icon" style="margin-right:3px;"></div></td><td width="115" id="bottom_help_title" align="left"><a style="font-weight: bolder;text-decoration:underline;cursor:pointer;" href="https://my.ecrs.com" target="_blank"><#Help#> & <#Support#></a></td>';
 
 	var tracing_path_Manual	= "/HelpDesk_Manual/?utm_source=asus-product&utm_medium=referral&utm_campaign=router";
-	var tracing_path_Utility = "/HelpDesk_Download/?utm_source=asus-product&utm_medium=referral&utm_campaign=router";
 	var model_name_supportsite = supportsite_model(based_modelid, odmpid, hw_ver);	// @ /js/support_site.js	
 			
-	footer_code += "<td width=\"310\" id=\"bottom_help_link\" align=\"left\">&nbsp&nbsp<a style=\"font-weight: bolder;text-decoration:underline;cursor:pointer;\" href=\"http://www.asus.com"+ href_lang +"Networking/"+ model_name_supportsite + tracing_path_Manual +"\" target=\"_blank\"><#Manual#></a>&nbsp|&nbsp<a style=\"font-weight: bolder;text-decoration:underline;cursor:pointer;\" href=\"http://www.asus.com"+ href_lang +"Networking/" + model_name_supportsite + tracing_path_Utility +"\" target=\"_blank\"><#Utility#></a>";
+	footer_code += "<td width=\"310\" id=\"bottom_help_link\" align=\"left\">&nbsp&nbsp<a style=\"font-weight: bolder;text-decoration:underline;cursor:pointer;\" href=\"http://www.asus.com"+ href_lang +"Networking/"+ model_name_supportsite + tracing_path_Manual +"\" target=\"_blank\"><#Manual#></a>&nbsp;|&nbsp;<a style=\"font-weight: bolder;text-decoration:underline;cursor:pointer;\" target=\"_blank\" href=\"https://github.com/ECRS/asuswrt-merlin\">Source</a>";
 
 	if(dsl_support && feedback_support){
 		footer_code += '&nbsp|&nbsp<a id="fb_link" href="/Advanced_Feedback.asp" target="_self" style="font-weight: bolder;text-decoration:underline;cursor:pointer;"><#menu_feedback#></a>';
